Tests/MongoSwiftTests/SDAMMonitoringTests.swift

```diff
@@ -56,6 +57,15 @@ final class SDAMTests: XCTestCase {
 
         center.removeObserver(observer)
 
+        var error = bson_error_t()
+        guard let uri = mongoc_uri_new_with_error(XCTestCase.connStr, &error) else {
+            throw MongoError.invalidUri(message: toErrorString(error))
+        }
+
+        guard let hostlist = mongoc_uri_get_hosts(uri) else {
+            throw MongoError.invalidResponse()
```

Comment:
Should we use a more descriptive error here, or is it clear from this enum case why the test might fail? I'm just contrasting this with the `invalidUri` error above, which includes a descriptive message.

Granted, I'm also not familiar with these tests.

Reply:
The above case has the fortune of having an error returned from the `mongoc` function, which lets you actually say something meaningful about why something went wrong. In this case however, we don't really know why `mongoc` did not give us something back, so the best we can do is say something along the lines of "Getting the hostlist did not work".

Also, `invalidResponse()` seemed most appropriate, and unfortunately does not take a `message` argument.  :( Otherwise, I'd have put in at least some kind of message about the `hostlist` not working out.

Reply:
didn't think of this before, but we should probably just call `XCTFail` here with a descriptive message rather than throwing.

Reply:
Great, done.

I've kept the [other throw](https://github.com/mongodb/mongo-swift-driver/pull/135/files/ba5716babeccbc766f6a0be78f2033f35deeb336#diff-bf6491c15a31640b2ab0c3bbf0fb4494R62), since that has a descriptive message and that seems to satisfy the principal goal of this discussion.

But since I'm not sure if throwing in general is something we want to avoid in Swift tests, I'll refrain from merging this until we're sure that we're okay with throwing a descriptive error for the above case. Let me know if we should use `XCTFail()` again instead.

Reply:
I think the actual error is fine, but also support changing to XCTFail if @jmikola cares.

Reply:
Sounds like a plan! @jmikola Let us know what you think.

Reply:
Done. Now they both trigger `XCTFail()`. I'll merge once Travis is green.